### ISSUE TITLE
Keyboard accessibility - Menu-Focus when tabbing is not visible (unperceivable)

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#101]
IE9 detail seite

When tabbing the highlighting of the main navigation menu element is not visible

Original
![image](https://cloud.githubusercontent.com/assets/7510842/3188826/5128fbea-ecbe-11e3-8a8a-d0b2f198a4fb.png)

New
There's no visible focus
![image](https://cloud.githubusercontent.com/assets/7510842/3188855/959b34e6-ecbe-11e3-8a81-63ac692c8346.png)
